torch/_torch_docs.py

```diff
@@ -736,11 +736,11 @@ def parse_kwargs(desc):
 
 add_docstr(torch.cat,
            r"""
-cat(seq, dim=0, out=None) -> Tensor
+cat(seq, dim=0, pad=False, pad_value=0, out=None) -> Tensor
 
 Concatenates the given sequence of :attr:`seq` tensors in the given dimension.
 All tensors must either have the same shape (except in the concatenating
-dimension) or be empty.
+dimension) or be empty. Otherwise, :attr:`pad` must be set true to force cat.
```

Comment:
A couple things here:
1) We should remove the part about "or be empty".  This is legacy behavior that people shouldn't rely on (this is not related to your PR, but since you are touching it, it would be good to fix up).
2) It's not clear to me what "to force cat" means or how the padding is done.  I would just split this paragraph in two.  The first says: "If padding is set to false,...", the second "If padding is set to true..." and explain those cases separately.